lib/task-data/tasks/dell-racadm-get-firmware-list-catalog.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'dell racadm get firmware list catalog',
+    injectableName: 'Task.Dell.Racadm.GetFirmwareListCatalog',
+    implementsTask: 'Task.Base.Dell.Racadm.Catalog',
+    optionsSchema: 'dell-racadm-catalog.json',
+    options: {
+        action: 'getSoftwareList'
```

Comment:
it looks the action 'getSoftwareList' and the task name is 'GetFirmwareLIst' would cause confusion, how do you think?